core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -310,6 +310,9 @@ object TopicCommand extends Logging {
     val describeOpt = parser.accepts("describe", "List details for the given topics.")
     val topicOpt = parser.accepts("topic", "The topic to be create, alter or describe. Can also accept a regular " +
                                            "expression except for --create option")
+    val topicOpt = parser.accepts("topic", "The topic to create, alter, describe or delete. It also accepts a regular " +
```

Comment:
this line is same as above line. compilation is failing due to duplicate ```val topicOpt```

Reply:
Aah thanks. I thought I resolved that conflict before pushing the change. Anyway I'll update this later today. Thanks!